src/graphql.ts

```diff
@@ -74,6 +75,7 @@ export const schema = urlDirective.transformer(
               paddle.typeDefs,
               njord.typeDefs,
               organizations.typeDefs,
+              profileAutocomplete.typeDefs,
```

Comment:
What queries you image having in this new schema, kinda feels to specific to be generic, maybe call it userExperience schema so it can have other stuff related to user experience profile and so on?

Reply:
No strong preference just observation.

Reply:
Fine to me ✔️